.github/workflows/image-transfer.yml

```diff
@@ -41,6 +41,16 @@ jobs:
           python3 -m pip show docker  # Verify installation
           sudo python3 vidivi.py push
 
+      - name: commit and push imagetransfer.log
+        run: |
+          cd release/vidivi/logs/
+          sudo mv vidivi-*.log imagetransfer.log
```

Comment:
please remove this line as it is not required

Reply:
log file will get create as vidivi-24-06-2024-10:05:07.log
so each time it get push new log file will be added
to avoid this moving to vidivi.log which will be override and push.

Reply:
As discussed, please update the log file https://github.com/Rakshithb1/release-script/blob/89d6e7735836306e77a1c5e98923cc6d03a264b5/release/vidivi/vidivi.py#L140

from `logFile = "vidivi-" + datetime.now().strftime("%d-%m-%Y-%H:%M:%S") + ".log"`
to `logFile = "vidivi.log"

Reply:
updated